### PR TITLE
fix mailing list for next 2.x branch

### PR DESCRIPTION
**Pull request by ebarboni**

### CODE REVIEW

File: archiva-modules/archiva-web/archiva-web-common/src/main/resources/org/apache/archiva/security.properties [hunk 1]
```diff
@@ -24,4 +24,4 @@
 email.validation.subject=Welcome to Archiva
 
 # Feedback page
-email.feedback.path=http://archiva.apache.org/mail-lists.html
+email.feedback.path=http://archiva.apache.org/mailing-lists.html
```
Comment (olamy) [inline archiva-modules/archiva-web/archiva-web-common/src/main/resources/org/apache/archiva/security.properties:27]: https?

File: archiva-modules/archiva-web/archiva-webapp/src/main/resources/org/apache/archiva/redback-security.properties [hunk 2]
```diff
@@ -24,6 +24,6 @@
 email.validation.subject=Welcome to Archiva
 
 # Feedback page
-email.feedback.path=http://archiva.apache.org/mail-lists.html
+email.feedback.path=http://archiva.apache.org/mailing-lists.html
```
Comment (olamy) [inline archiva-modules/archiva-web/archiva-webapp/src/main/resources/org/apache/archiva/redback-security.properties:27]: https?

**Review by olamy — CHANGES_REQUESTED** [review]
while we are here maybe we should change to https as well?

File: archiva-modules/archiva-web/archiva-web-common/src/main/resources/org/apache/archiva/security.properties (hunk 1 above)
Comment (olamy) [inline archiva-modules/archiva-web/archiva-web-common/src/main/resources/org/apache/archiva/security.properties:27]:
```suggestion
email.feedback.path=https://archiva.apache.org/mailing-lists.html
```

File: archiva-modules/archiva-web/archiva-webapp/src/main/resources/org/apache/archiva/redback-security.properties (hunk 2 above)
Comment (olamy) [inline archiva-modules/archiva-web/archiva-webapp/src/main/resources/org/apache/archiva/redback-security.properties:27]:
```suggestion
email.feedback.path=https://archiva.apache.org/mailing-lists.html
```

**Review by olamy — CHANGES_REQUESTED** [review]

**Comment by ebarboni** [comment]
https used now and rebased.

**Pull request merged**